SQL/0000-00-00-schema.sql

```diff
@@ -652,6 +659,46 @@ CREATE TABLE `mri_protocol_checks` (
   PRIMARY KEY (`ID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+
+-- ********************************
+-- BIDS tables
+-- ********************************
+
+CREATE TABLE `BIDS_category` (
+ `ImagingCategory` varchar(255) NOT NULL PRIMARY KEY
```

Comment:
shouldn't there be an ID column in that new table? @PapillonMcGill

Reply:
also varchar(225) seems a bit much for that column.

Reply:
I added an auto-increment ID column, again just in case @PapillonMcGill says I should (so we have both commits; one without and the last one with)